packages/client/machines/appMusicPlaylistsMachine.ts

```diff
@@ -114,7 +137,9 @@ export const appMusicPlaylistsModel = createModel(
             ) => args,
             ///
 
-            REMOVE_ACTOR_FROM_PLAYLIST_LIST: (args: { roomID: string }) => args,
+            STOP_AND_REMOVE_ACTOR_FROM_PLAYLIST_LIST: (args: {
+                roomID: string;
+            }) => args,
```

Comment:
I have to say that this event is really interesting Whenever you want to (stop and) remove an actor, you send the event to the machine itself. This is really elegant.